fetch-mock/index.d.ts

```diff
@@ -104,33 +104,41 @@ interface MockOptions {
      * as specified above
      */
     response?: MockResponse | MockResponseFunction;
+    /**
+     * integer, n, limiting the number of times the matcher can be used.
+       If the route has already been called n times the route will be
+       ignored and the call to fetch() will fall through to be handled by
+       any other routes defined (which may eventually result in an error
+       if nothing matches it).
+     */
+    times?: number;
 }
 
 type MockCall = [string, MockRequest];
 
 interface MatchedRoutes {
-    matched: Array<MockCall>;
-    unmatched: Array<MockCall>;
+    matched: MockCall[];
```

Comment:
tslint again:
`array-type` - Array type using 'Array<T>' is forbidden for simple types. Use 'T[]' instead.`